models/platform-limits/src/product-discounts/builder.spec.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import { TProductDiscountLimitsProjection } from './types';
+import * as ProductDiscountLimitsProjection from './index';
+
+describe('building', () => {
+  it(
+    ...createBuilderSpec<
+      TProductDiscountLimitsProjection,
+      TProductDiscountLimitsProjection
+    >(
+      'default',
+      ProductDiscountLimitsProjection.random(),
```

Comment:
It would be great to use the preset to generate a model instance and assert the `totalActive` property also has a couple of the expected attributes.

Reply:
Done https://github.com/commercetools/test-data/pull/657/commits/6bfab6455c8b504b50e7e12d8b8c3e1da173dcef#diff-c5cb7d76dd373d32885396efb72c25d87c7d548a6671dba26273537a306cdfa4